src/web/api/src/main/java/org/locationtech/geogig/web/api/CommandBuilder.java

```diff
@@ -269,6 +269,11 @@ static BranchWebOp buildBranch(ParameterSet options) {
         BranchWebOp command = new BranchWebOp();
         command.setList(Boolean.valueOf(options.getFirstValue("list", "false")));
         command.setRemotes(Boolean.valueOf(options.getFirstValue("remotes", "false")));
+        command.setBranchName(options.getFirstValue("branchName", null));
+        // TODO: enable these options if necessary
+        //command.setAutoCheckout(Boolean.valueOf(options.getFirstValue("autoCheckout", "false")));
```

Comment:
@emerkle826 right, I don't think we want autocheckout on the server side, since every op should be running inside a transaction, chances are when you commit the transaction, the main repo's HEAD gets checked out hence changing the "default" view of the repo...